### PR TITLE
#3077 Hide disambiguation items from depiction search 

### LINKED ISSUE #3077: Hide disambiguation items from depiction search
This first item is a disambiguation item, it should not be shown:
Screenshot from 2019-08-28 12-09-18
There are other kinds of "technical" items that probably should not be shown either, for instance Wikimedia categories like https://www.wikidata.org/wiki/Q32415105

### PR DESCRIPTION
**Pull request by macgills**
**Description (required)**

Fixes #3077 

What changes did you make and why?
- when fetching depictions get the entity instead of the search result

**Tests performed (required)**

Tested ProdDebug on Nexus5x with API level 27

**Screenshots (for UI changes only)**

image

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/wikidata/WikidataDisambiguationItems.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package fr.free.nrw.commons.wikidata
+
+
+enum class WikidataDisambiguationItems(val id: String) {
+    DISAMBIGUATION_PAGE("Q4167410"), INTERNAL_ITEM("Q17442446"), CATEGORY("Q4167836");
```
Comment (ashishkumar468) [inline app/src/main/java/fr/free/nrw/commons/wikidata/WikidataDisambiguationItems.kt:5]: Should we add comments stating why we choose these constants?

Comment (macgills) [reply app/src/main/java/fr/free/nrw/commons/wikidata/WikidataDisambiguationItems.kt:5]: I could add a link to [this comment](https://github.com/commons-app/apps-android-commons/issues/3077#issuecomment-625601555)? I don't think it is much deeper than "these are disambiguation items" which is what the classname says

Comment (nicolas-raoul) [reply app/src/main/java/fr/free/nrw/commons/wikidata/WikidataDisambiguationItems.kt:5]: I think adding this link is good enough indeed.

**Comment by nicolas-raoul** [comment]
I installed the latest macgills/3077-hide-disambiguation-items and strangely I am still seeing https://www.wikidata.org/wiki/Q424946 which is an instance of https://www.wikidata.org/wiki/Q4167410 :
Screenshot_20200512-185749_Commons
Any idea what might be the problem?

**Comment by macgills** [comment]
@nicolas-raoul I misunderstood the scope, going off your first picture I thought this was for during upload only. Is it alright to make a new ticket to support this functionality on that screen?

**Comment by nicolas-raoul** [comment]
Oh, my bad! The issue is indeed for upload only. Showing disambig pages in Explore is OK.

File: app/src/main/java/fr/free/nrw/commons/wikidata/WikidataDisambiguationItems.kt (hunk 1 above)
Comment (ashishkumar468) [reply app/src/main/java/fr/free/nrw/commons/wikidata/WikidataDisambiguationItems.kt:5]: Yes that should be fine

**Pull request merged**